src/server/graphql/queries/organization.js

```diff
@@ -29,11 +29,12 @@ export default {
 
     const inactiveUserCount = orgUsers.filter((user) => user.inactive).length;
     const activeUserCount = orgUsers.length - inactiveUserCount;
-
     return {
       ...org,
-      inactiveUserCount,
-      activeUserCount
+      orgUserCount: {
```

Comment:
i put these in a sub-object so both are resolved in 1 lookup instead of 2

Reply:
Seems safe to try...